tests/main/degraded/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Check that the system is not in "degraded" state
+
+# run this early to ensure no test created failed units yet
+priority: 500
+
+execute: |
+    #shellcheck source=tests/lib/systemd.sh
+    . "$TESTSLIB"/systemd.sh
+    wait_for_service "multi-user.target"
+
+    if systemctl status | grep "State: [d]egraded"; then
```

Comment:
Why the [d]egraded regex?

Reply:
systemctl status will print the process tree. So `systemctl status|grep foo` will include:
```
18012 grep foo
```
in the output. With the `grep [f]oo` grep will search for the string `foo` so the grep command itself (which is grep [f]oo) is no longer included in the output.

Reply:
Ah, smart, thanks!